geode-core/src/main/java/org/apache/geode/distributed/internal/tcpserver/TcpServer.java

```diff
@@ -77,9 +77,9 @@
    * <p>
    * This should be incremented if the gossip message structures change
    * <p>
-   * 1000 - gemfire 5.5 - using java serialization 1001 - 5.7 - using DataSerializable and
```

Comment:
I didn't check but that's my assumption.  It likes the comment with the HTML breaks in place.